### PR TITLE
bump dependencies without breaking changes

### LINKED ISSUE #2333: Review and update data README
**Is your feature request related to a problem? Please describe.**
There is a README about our test data at https://github.com/bbc/simorgh/blob/latest/data/README.md however it is currently not very clear. It also needs more information about the assets we have created for audio-video content.

We should review this document and update it so that it's clear, accurate and comprehensive. This will make life easier for testers and developers to avoid having to ask around for test IDs/URLs.

**Describe the solution you'd like**
As above. In particular:

- TBD

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves #7685, #7674, #7662, #7659, #7654, #7653, #7651 

**Overall change:**
_bumps dendencies._

**Code changes:**

- _bumps dendencies._

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [X] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/Include/amp/VjAmp.jsx [hunk 1]
```diff
@@ -75,7 +75,8 @@ const StyledAmpIframe = styled(AmpIframe)`
       cursor: pointer;
       display: block;
       padding: ${GEL_SPACING} ${GEL_SPACING_DBL};
-      &:hover, &:focus {
+      &:hover,
+      &:focus {
```
Comment (karinathomasbbc) [inline src/app/containers/Include/amp/VjAmp.jsx:79]: Is this intentional?

Comment (rhenshaw56) [reply src/app/containers/Include/amp/VjAmp.jsx:79]: yes, the bump to prettier resulted in some linting errors.

Comment (karinathomasbbc) [reply src/app/containers/Include/amp/VjAmp.jsx:79]: Sorry! This looked like an addition, but it's just whitespace! All good, thanks 👍

**Review by karinathomasbbc — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -135,7 +135,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-helmet": "^6.1.0",
-    "react-lazyload": "^2.6.8",
+    "react-lazyload": "^3.0.0",
```
Comment (joshcoventry) [inline package.json:138]: @rhenshaw56 Are there any breaking changes in this major version bump?  I can see updated tests but wanted to check if anything else has changed.

Comment (rhenshaw56) [reply package.json:138]: @joshcoventry I spotted no breaking changes. The snapshot updates was as a result of the vendor appending `classnames` to the component wrapper [see commit](https://github.com/twobin/react-lazyload/commit/0b042b5d7656f45ec44f91ac76122f2a32e11704)

**Review by chewu — APPROVED** [review]

**Pull request merged**